### PR TITLE
Fix crash when launching applications on iOS device in debug mode

### PR DESCRIPTION
**Pull request by Fatme**
Fixes https://github.com/NativeScript/nativescript-cli/issues/1362

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by teobugslayer** [comment]
:+1:

File: mobile/ios/ios-device.ts [hunk 1]
```diff
@@ -54,6 +54,21 @@ export class IOSDevice implements Mobile.IiOSDevice {
 			this.deviceInfo.version = this.getValue("ProductVersion");
 			this.deviceInfo.color = this.getValue("DeviceColor");
 			this.deviceInfo.isTablet = productType && productType.toLowerCase().indexOf("ipad") !== -1;
+
+			productType = productType.toLowerCase().trim();
+			let majorVersionAsString = _.last(productType.split(",")[0]);
```
Comment (Fatme) [inline mobile/ios/ios-device.ts]: It is better to use regex here in order to works correctly if productType is for example `iphone10,2`

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**